src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator start work");

```

Comment:
The message 'Excavator start work' could be more informative. It should indicate that the Excavator is working, for example: 'Excavator is now working.'